core/domain/activity_services.py

```diff
@@ -39,6 +48,15 @@ def update_featured_activity_references(featured_activity_references):
 
     This assumes that the input is a list of activity references corresponding
     to publicly-viewable activities.
+    
+    Args:
+        featured_activity_references: A list of featured activity references
+    
+    Returns:
+        None
+    
+    Raises:
+        Exception:checking if an activity reference list does not have duplicates.
```

Comment:
Follow format in style guide: single space after colon, capitalize part after colon.

In addition, state the failure criterion (again, cf. the style guide):
```
    Exception: The input list of ActivityReference domain objects has duplicates.
```